next.config.js

```diff
@@ -0,0 +1,9 @@
+const { i18n } = require('./next-i18next.config')
+
+/** @type {import('next').NextConfig} */
```

Comment:
do we really need /** @type {import('next').NextConfig} */ ?

Reply:
discussed